File: RAI/Clase13/Python/Pollo.py
from Animal import Animal

class Pollo(Animal):

    def __init__(self, nombre: str, familia: str, especie: str, color_plumas:str):
        super().__init__(nombre, familia, especie)
        self.color_plumas = color_plumas

    def poner_huevo(self):
        print(self.nombre + " pone huevos ")

File: RAI/Clase13/Java/Pollo.java
//package RAI.Clase13.Java;

public class Pollo extends Animal{
    private String colorPlumas;

    public Pollo(String nombre, String familia, String especie, String colorPlumas){
        super(nombre, familia, especie);
        this.colorPlumas = colorPlumas;
    }

    public void poneHuevo(){
        System.out.println(this.nombre + "Ha puesto un huevo");
    }

    @Override
    public void haceSonido(){
        System.out.println(this.nombre + " hace Piopio...");
    }
    
}
